### DISCUSSION TITLE
ROI Group Analysis MNI

### DISCUSSION THREAD

**Discussion by frankatimm** [simnibs/simnibs#327] [9. Results and analysis]
Hi everybody,

Can anyone tell me if there are instructions available on how to run an ROI Group Analysis in MNI space. Somehow I can only find instructions for single-subject ROI Analysis or group analysis in FsAverage space. Thank you in advance for any help.

Best,
Franka

**Comment by oulap** [comment]
Hi Franka,

When you do the simulation (either in the GUI or using a script) set the "map_to_mni" variable to true. If you are doing the simulations in the GUI, you should tick the box named "Transform to MNI space", see here:

https://simnibs.github.io/simnibs/build/html/documentation/gui.html#simulation-options-window

If you are using matlab or python scripts, here's an example of how to set the "map_to_mni" variable to true:

https://simnibs.github.io/simnibs/build/html/tutorial/advanced/group_analysis.html#python

The simulation results will then be transformed to the MNI template space and from there you can analyze the e-field results with whatever atlas or target ROI you want.

**Reply by frankatimm** [reply]
Ok, thank you so much. I am still not sure how I find the name for my target ROI, in the example defined as 'lh.4'. In my case I want to look at the left and right DLPFC. Do you know where can I look this up? Thank you in advance for your help.

Best, 
Franka

**Reply by oulap** [reply]
You need to find an atlas, which has the ROI you're interested in. Once you find that atlas (in MNI space) you can just extract the label you're interested in from the atlas, which will give you a binary mask of the region. Because the atlas and the e-field results are in the same space you can then just apply that mask to get the e-fields within the ROI. If you need pointers on where to find atlases, lead-dbs has a nice list of different ones:

https://www.lead-dbs.org/helpsupport/knowledge-base/atlasesresources/cortical-atlas-parcellations-mni-space/

**Comment by ghost** [comment]
Hello, I did some simulations with simnibs and I want to do ROI analysis. ROI is the frontal lobe, but I can not find any atlas with that as a label. also, simnibs documentation only suggests three atlases :

https://simnibs.github.io/simnibs/build/html/documentation/python_lib/auto/simnibs.subject_atlas.html#simnibs.subject_atlas

How would I do this analysis?

**Reply by jduemose** [reply]
Hi you will either need to find a different atlas that has the desired parcellation (i.e., including the frontal lobe) or combine the labels in an existing atlas to roughly correspond to the ROI you are after. The ROI is just a binary mask created from an atlas. As Oula mentioned above, once you have loaded an atlas (python or matlab), you are free to combine labels into a binary mask.

Perhaps #364 contains useful information?

There is also a section on [this](https://surfer.nmr.mgh.harvard.edu/fswiki/CorticalParcellation) page from FreeSurfer about lobe mapping which might be helpful.

Hope this helps,
Jesper
